docs/community_resources/release_process.md

```diff
@@ -0,0 +1,62 @@
+# eZ Platform release process and roadmap
+
+## Release process
+
+eZ Platform has two distributions:
+
+- eZ Platform is an open source Content Management System (CMS) developed by eZ Systems together with the open source community. eZ Platform's code base is available on GitHub under the GPLv2 license. eZ Platform comes with no commercial support and maintenance services. It is supported by the community on public channels.
+- eZ Platform Enterprise Edition (EE) is commercial software available under eZ Business User License (BUL) to eZ Enterprise subscribers. It is comprised of eZ Platform Open Source edition, additional enterprise support and maintenance services, as well as additional features which are not available in the open source software.
+
+We manage the release of eZ Platform using an agile iterative process and a continuous software development model, which is why we provide two kinds of eZ Platform releases:
+
+- Long Term Support releases (LTS) which are supported by eZ Systems for a long period of time (see [support lifecycle below](#support-lifecycle)), for eZ Enterprise subscribers.
+- Fast Track releases (FT) give access to the latest features and are supported for a short period of time, only until the next FT release is introduced. These are supported for both the open source community and eZ Enterprise subscribers.
+
+FT releases are tailored for those who want to stay up-to-date with newest functionalities,
+while LTS releases are suitable for highly stable enterprise rollouts.
+
+We usually release eZ Platform four times a year following the seasons (winter, spring, summer and fall). This usually includes one LTS release and three FT releases.
+
+## Versioning conventions
+
+Both eZ Platform editions use [semantic versioning](http://semver.org/).  
+
+The version number of eZ Platform and all its internal components follows the semantic versioning conventions: vX.Y.Z.
+
+- Changes to X indicate a breaking change to the API, and developers know that they may need to implement a re-write of their code to continue using the API.
```

Comment:
Maybe rather
`- Changes to X indicate a breaking changes, and while it is usually mostly internal things that change, developers know that they may need to see our change logs if they need to adjust their code to continue using the API / Features. If there are larger breaks, like v2 with admin ui, this is announced well in advance of the upcoming release.`

WDYT @rolandbenedetti Maybe to verbose here, but point is to not imply API changes a lot for every major.